dev/package-examples/auditd-2.0.4/auditbeat/config/config.yml

```diff
@@ -0,0 +1,45 @@
+{{ if eq .GOOS "linux" -}}
+  {{ if .Reference -}}
+  # The auditd module collects events from the audit framework in the Linux
+  # kernel. You need to specify audit rules for the events that you want to audit.
+  {{ end -}}
+- module: auditd
+  {{ if .Reference -}}
```

Comment:
@andrewkroh Where is this one coming from?

Reply:
Ah, just answered it myself. You combined the reference config and this config into 1 👍